package.json

```diff
@@ -7,6 +7,7 @@
     "@percy/cli": "^1.0.0-beta.71",
     "@percy/playwright": "^1.0.1",
     "@playwright/test": "^1.17.2",
+    "@types/eventemitter3": "^1.0.0",
```

Comment:
Let's do the upgrade for event emitter in a different PR.

Reply:
This isn't upgrading eventemitter, it is only adding type definitions for the existing package so that types show up when viewing in VS Code.

I split this into two separate PRs:

- type annotations: https://github.com/nasa/openmct/pull/4789
- and this one

Reply:
I'll mar as resolved since this PR no longer has type updates (see other PR)